lib/MCToIRLifter.cpp

```diff
@@ -314,6 +341,41 @@ void MCToIRLifter::VisitInstruction(remill::Instruction &inst,
     }
   }
 
+  // Check to see if this location has type and value information associated with it
+  //inst.pc
```

Comment:
Single space after `//`. When referring to variables/code things, reference with backticks. e.g.
```
// `inst.pc`.
```